server/deadfish.cpp

```diff
@@ -86,4 +86,4 @@ CollisionMask::CollisionMask(const FlatBuffGenerated::CollisionMask* fb_Col) {
     fixtureDef.density = 1;
 	body->CreateFixture(&fixtureDef);
 	body->SetUserData(this);
-}
+}
```

Comment:
why you kill endline when endline good

Reply:
please review your changes before submitting a pr and delete meaningless changes like this

Reply:
No idea, probably been drunk or sth. That's weird af